super_team_picker/routes/cohorts.js

```diff
@@ -0,0 +1,131 @@
+const express = require('express');
+const knex = require('../db/client');
+
+const router = express.Router();
+
+//-------------- Index to List Cohorts
+router.get('/', (req,res) => {
+    knex('cohorts')
+    .orderBy('created_at', 'desc')
+    .then(cohorts => {
+      res.render('cohorts/index', {cohorts: cohorts})
+    })
+  })
+
+//-------------- Create New Cohorts
+router.get('/new', (req, res) => {
+    res.render('cohorts/new', { cohort: false });
+})
+
+router.post('/', (req, res) => {
+    knex('cohorts')
+    .insert({
+      logoUrl: req.body.logoUrl,
+      name: req.body.name,
+      members: req.body.members
+    })
+    .returning('*')
+    .then(cohorts => {
+      const cohort = cohorts[0]
+      res.redirect(`cohorts/${cohort.id}`)
+    })
+  })
+
+//-------------- Show a Single Cohort
+function shuffle(array) {
+  for (let i = array.length - 1; i > 0; i--) {
+    let j = Math.floor(Math.random() * (i + 1));
+    [array[i], array[j]] = [array[j], array[i]];
+  }
+  return array;
+}
+
+function assignTeams(method, quantity, members) {
+  let names = members.split(",");
+
+  names.forEach((name, index, names) => {
+    names[index] = name.trim();
+  });
+
+  names = shuffle(names);
+
+
+  let teams = [];
+  if (method == "TeamCount") {
```

Comment:
suggestion: you can also add a check and message for scenarios that cannot happen.
e.g. number of team members : 5
team count: 6
error message: please enter a valid number
something like that.

everthing looks good!!!